.buildkite/pipeline.yml

```diff
@@ -105,6 +106,7 @@ steps:
         plugins: *common_plugins
         artifact_paths:
           - "build/results/*"
+          - "build/results/crashes/*"
```

Comment:
Wait, those are in this test PR but not in the original PR! We need to add those in https://github.com/wordpress-mobile/WordPress-iOS/pull/20658 don't we?

_At the time I reviewed https://github.com/wordpress-mobile/WordPress-iOS/pull/20658 and https://github.com/wordpress-mobile/WordPress-iOS/pull/20649 I (incorrectly) assumed that `build/results/*` would traverse directories recursively and thus that they would include those `.ips` file, but I now realize I was mistaken and that either we need to add this like like you did, or we need to use some sort of `build/results/**/*` more eager notation._

Reply:
Ok, nvm, [they're there in the original PR too after all](https://github.com/wordpress-mobile/WordPress-iOS/pull/20649/files#diff-a3991ebf1475eb82acab13946ffc1eca02b43917f6d5bec3898f45c8b0b9bd53). It's late, maybe I should log off and get some rest 😅

Reply:
I've actually tried using `build/results/**/**` in this test PR here [`9c20ecb`](https://github.com/wordpress-mobile/WordPress-iOS/pull/20564/commits/9c20ecb9eee63281cb1b65ee1b54b03fe7de062a), but the result was not ideal. It ended up [displaying all the files within the `.xcresult` file](https://buildkite.com/automattic/wordpress-ios/builds/14106#01880159-5054-484d-b375-1ce7987bec55) (~200 files) and I couldn't actually see the crash logs listed.

I haven't tried `build/results/**/*` though. 🤔

Reply:
Ah yeah if the `.xcresult` file (which is really a directory) is still in `build/results/` after we zipped it, then even `build/results/**/*` will include it as well. I assumed that we wanted everything (recursively) that was in `build/results/`, while in fact we don't and instead only want the top-level files (like the `xcresult.zip` and the files in `crashes/`, so maybe it's better to be explicit like you did with `build/results/*`+`build/results/crashes/*` after all.